app/lib/hesa/code_sets/bursary_levels.rb

```diff
@@ -7,15 +7,38 @@ module BursaryLevels
       # https://www.hesa.ac.uk/collection/c22053/e/burslev
       # Note that Tier 1, Tier 2, and Tier 3 are fetched from the previous year, i.e:
       # https://www.hesa.ac.uk/collection/c20053/e/burslev
```

Comment:
are we up to date with https://www.hesa.ac.uk/collection/c23053/e/burslev ?

Reply:
Yup that's what triggered this PR. I should probably update the reference in that comment.